tests/test_scripts.py

```diff
@@ -0,0 +1,24 @@
+import os
+import runpy
+
+import pytest
+from pathlib import Path
+
+
+ROOT_DIR = Path(os.path.join(os.path.dirname(__file__), ".."))
```

Comment:
We can use `pathlib.Path()` even further by replacing this as follows:

```suggestion
ROOT_DIR = Path(__file__).parent.parent
```

which is cleaner!